### ISSUE TITLE
Issue with restarting of subjobs

### ISSUE THREAD

**Issue by antonf-ekb** [pyiron/pyiron_base#316]
Hi!
First of all, I'd like to thank the authors for the nice code for organizing simulation workflows. My query is not quite a bug report but rather an issue report. Suppose I have some job, which is based on many subjobs, i.e. phonopy, elastic, etc. If some of the subjobs of a parent job fail to converge, I restart them using `project.load("subjob_name").restart()`. The subjobs restart by means of creating new subjobs with "_restart" suffix. But when after the restarted jobs converged and I run `parent_job.collect_output()`, the results of this parent job do not change, like Pyiron did not read out the data from the restarted subjobs. 
If I try to run `project.load("subjob_name_restart").collect_output()`  it reports the error "Either the OUTCAR or vasprun.xml files need to be present" although the files exist in the corresponding directory. If I run `project.load("subjob_name").collect_output()` it reports "WARNING - Unable to parse the vasprun.xml file. Will attempt to get data from OUTCAR" i.e. it reads the old unconverged job data.
So, summing up, what is the correct way to restart subjobs and read their output afterward?

**Comment by sudarsan-surendralal** [comment]
This issue may not be related to VASP in pyiron but I'll give it a shot anyway. The way you restart a new job from the non-converged child job is correct. It's only that the new job is unrelated to the parent job so even if this one converges, it doesn't change anything in the parent job. If the problem is convergence, the cleanest solution is to re-run the jobs by increasing the number of ionic steps (`NSW` parameter in VASP). 

However, for your specific problem, can you try the following:

```
job_non_conv = project.load("subjob_name")
job_non_conv.structure = job_non_conv.get_structure()
job_non_conv.run(delete_existing_job=True)
```

This re-runs the child job with the updated structure. Then after this, try:

```
parent_job.status.refresh = True
parent_job.run()
```

But since this modifies the input of the child job, IMO, this solution isn't recommended! @jan-janssen is there a better way to handle this?

**Comment by antonf-ekb** [comment]
Thank you, Sudarsan, your suggestion worked out!
